starters/next13-zustand-bulma/src/app/api-example/page.tsx

```diff
@@ -19,7 +32,10 @@ export default function ApiExamplePage() {
         >
           API Example: Fetch Data using API route
         </h1>
-        <div className="block px-5">{/* Add Greeting Component Here */}</div>
+        <div className="block px-5">
+          {/* @ts-expect-error Async Server Component - please read: https://beta.nextjs.org/docs/data-fetching/fetching#asyncawait-in-server-components for more info */}
+          <Greeting greeting={greeting}/>
```

Comment:
We should probably give them a link, or a hint or something to show they can use the `?greeting=` query param on this page